### DISCUSSION TITLE
Reopening of the discussion tab

### DISCUSSION THREAD

**Discussion by caring-coder** [verronpro/office-stamper#296] [Announcements]
I reopen the discussion tab, as i feel the codebase is getting mature enough to discuss reasonably about.
Especially the clear difference between core library code, and library API for developper not maintaining the library itself.

**Comment by StefanLobbenmeierObjego** [comment]
Somehow it is also possible to discuss new releases:
![image](https://github.com/verronpro/docx-stamper/assets/122794560/0a8d6be1-9ba3-4f4d-80a9-e79801c1a7b6)

I did not even know that is a GitHub feature that existed, but that is really nice :smile:

**Reply by caring-coder** [reply]
Yes, i feel it's a good place to aggregate feedback, without always going trough issues. 
Hopefully when there is enough information gathered, it could be sorted into READMEs files or anything else that makes sense.
